test/old-storage

```diff
@@ -98,8 +98,19 @@ for oldtar in $oldtars; do
     export CH_IMAGE_STORAGE=$storage
     INFO "unpacked: $(du -sh "$storage" | cut -f1) bytes in $(du --inodes -sh "$storage" | cut -f1) inodes"
 
+    case ${storage#*-} in
+        0.29|0.30|0.31)
+            INFO "working around issue #1662"
```

Comment:
isn't #1662 and PR and not an issue. (took me a while to figure that one out). Maybe rename to "working around issue in PR #1662"